packages/nextjs/components/scaffold-eth/Contract/WriteOnlyFunctionForm.tsx

```diff
@@ -93,6 +95,21 @@ export const WriteOnlyFunctionForm = ({
   });
   const zeroInputs = inputs.length === 0 && abiFunction.stateMutability !== "payable";
 
+  const handleCopyCalldata = async () => {
+    try {
+      const calldata = encodeFunctionData({
+        abi: abi,
+        functionName: abiFunction.name,
+        args: getParsedContractFunctionArgs(form),
+      });
+      await navigator.clipboard.writeText(calldata);
+      notification.success("Calldata copied to clipboard!");
```

Comment:
I think its fine if we don't show notification and have the similar UX as other copy to clipboard elements on page. 

Checkout https://github.com/BuidlGuidl/abi.ninja/pull/179/files#r2052388958, so let's remove the notification from here. We can show notification when there is errro 🙌